### ISSUE TITLE
genimage warning: brctl not found

### ISSUE THREAD

**Issue by themisa** [clustervision/trinity#278]
* r3
0d12eb5

warning after executing:
[root@controller ~]# genimage -o centos7.0 -a x86_64 -p trinity

```
/install/custom/netboot/centos/trinity.centos7.postinstall: line 74: brctl: command not found
```

**Comment by hansthen** [comment]
Seems to be the same problem as in: #281

**Comment by hansthen** [comment]
I think I accidentally solved this in ca7daf583aa3197581f2781e4611d27f1ebed01c. Assigning to Themis for testing.

**Comment by themisa** [comment]
fixed in * r3
06c3594
